apps/console/src/features/organization-discovery/components/edit-organization-discovery-domains.tsx

```diff
@@ -275,9 +275,7 @@ const EditOrganizationDiscoveryDomains: FunctionComponent<EditOrganizationDiscov
                                     </Hint>
                                 ) }
                                 component={ TextFieldAdapter }
-                                InputProps={ {
-                                    readOnly: true
-                                } }
+                                readOnly
```

Comment:
Shouldn't we make this true/false based on the scopes/roles?

Reply:
This organization name shouldn't be changed.